Consolidate and cross-link kubecf documentation. (#87)
SUSE/kubecf: doc/dev/minikube.md

# Use the password from the previous step when requested.
cf auth -u admin -p "${acp}"
```

### Advanced Topics

#### Diego vs Eirini

Diego is the standard scheduler used by kubecf to deploy CF

> applications. Eirini is an alternative talking more directly to the

Reviewer: I added a comment regarding the same phrase above.

Author: Fixed with commit 9918b20dcb257d8865a289cee9653b710a4662a0.

underlying Kube cluster.

> To activate this alternative add a file matching the pattern

Reviewer: ```suggestion
To activate this alternative, add a file matching the pattern
```

Author: Fixed with commit 9918b20dcb257d8865a289cee9653b710a4662a0.

`*values.yaml` to the directory __dev/kubecf__ and containing

```yaml
features:
  eirini:
    enabled: true
```

before deploying kubecf.

#### Ingress

> By default the cluster is exposed through its kubernetes services.

Reviewer: ```suggestion
By default, the cluster is exposed through its Kubernetes services.
```

Author: Fixed with commit 9918b20dcb257d8865a289cee9653b710a4662a0.

To use the NGINX ingress instead, it is necessary to:

  - Install and configure the Nginx Ingress Controller.
  - Configure kubecf to use ingress.

This has to happen before deploying kubecf.

##### Installation of the NGINX Ingress Controller

```sh
helm install stable/nginx-ingress \
  --name ingress \
  --namespace ingress
  --set "controller.service.externalIPs={$(minikube ip)}"
```

> The last option above assigns the external IP of the cluster to the

Reviewer: ```suggestion
The last flag in the command above assigns the external IP of the cluster to the
```

Author: Fixed with commit 9918b20dcb257d8865a289cee9653b710a4662a0.
